test/functional/adb-commands-e2e-specs.js

```diff
@@ -151,6 +151,17 @@ describe('adb commands', function () {
     await adb.setAnimationState(true);
     (await adb.isAnimationOn()).should.be.true;
   });
+  it('should be able to set device locale via setting app @skip-ci', async () => {
+    // Operation not allowed: java.lang.SecurityException: Package io.appium.settings has not requested permission android.permission.CHANGE_CONFIGURATION
+    // is shown if the setting apk is not updated.
+    await adb.grantPermission('io.appium.settings', 'android.permission.CHANGE_CONFIGURATION');
+
+    await adb.setDeviceSysLocaleViaSettingApp('fr', 'fr');
+    (await adb.getDeviceSysLocale().should.equal('fr-FR'));
```

Comment:
I assume only `await adb.getDeviceSysLocale()` should be enclosed into parentheses. Same thing below.